internal/resources/policy/type/security/input_schema.go

```diff
@@ -16,21 +17,24 @@ import (
 	reciperesource "github.com/vmware/terraform-provider-tanzu-mission-control/internal/resources/policy/type/security/recipe"
 )
 
-var inputSchema = &schema.Schema{
-	Type:        schema.TypeList,
-	Description: "Input for the security policy, having one of the valid recipes: baseline, custom or strict.",
-	Required:    true,
-	MaxItems:    1,
-	MinItems:    1,
-	ForceNew:    true,
-	Elem: &schema.Resource{
-		Schema: map[string]*schema.Schema{
-			reciperesource.BaselineKey: reciperesource.Baseline,
-			reciperesource.CustomKey:   reciperesource.Custom,
-			reciperesource.StrictKey:   reciperesource.Strict,
+var (
+	inputSchema = &schema.Schema{
+		Type:        schema.TypeList,
+		Description: "Input for the security policy, having one of the valid recipes: baseline, custom or strict.",
+		Required:    true,
+		MaxItems:    1,
+		MinItems:    1,
+		ForceNew:    true,
+		Elem: &schema.Resource{
+			Schema: map[string]*schema.Schema{
+				reciperesource.BaselineKey: reciperesource.Baseline,
+				reciperesource.CustomKey:   reciperesource.Custom,
+				reciperesource.StrictKey:   reciperesource.Strict,
+			},
 		},
-	},
-}
+	}
+	recipesAllowed = [...]string{reciperesource.BaselineKey, reciperesource.CustomKey, reciperesource.StrictKey}
```

Comment:
why do we need [...]string{}, shouldn't it be []string{}

Reply:
Without this we will need to specify the size of the array.

Initially it was a slice but since the requirement is of a constant immutable slice so we have made it an array.

The ... will help in adding more values to the array without changing anything in the LHS.

Reply:
Even with []string{} we need not specify the size of the array

Reply:
Yes, a slice will also work but we don't have a requirement to modify this list during runtime. So an array might be more suited for this use case.